collections/climate-change-initiative.yaml

```diff
@@ -0,0 +1,83 @@
+Name: climate_change_initiative

```

Comment:
We could consider a more specific name for the dataset, without underscores, like "Essential Climate Variables (ECV)".